### PR TITLE
Nostromo maintenance

### PR DESCRIPTION
**Pull request by su8044**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by su8044** [comment]
Adding Nostromo to QMK

**Comment by tzarc** [comment]
Please remove the design files and place them in a separate repository.

**Comment by su8044** [comment]
Will do.

**Comment by su8044** [comment]
Alright, i left my master branch without the design files and the nostromo_mainteinance branch with them. is this correct? if not, could you please point me to the correct way? i searched the docs on the contributions but that was one of those things that i didn't quite get. thanks!

**Comment by su8044** [comment]
if it is the other way around (I'm starting to think it is), how do i do that on git? I'm quite noob on the subject

File: keyboards/nostromo/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = STM32F411
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/nostromo/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```

File: keyboards/nostromo/rules.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+ENCODER_ENABLE = no         # Enable rotary encoder support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/nostromo/rules.mk:23]:
```suggestion
```

File: keyboards/nostromo/info.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
+{
+    "keyboard_name": "nostromo",
+    "url": "https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo",
+    "maintainer": "su8044",
+    "width": 12,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/nostromo/info.json:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Design files still need to be removed.

**Comment by su8044** [comment]
Guys i'm having trouble isntalling MSYS and even if i isntall it i have no idea how to easily swap the location of the files. could you please give me a hand?

**Comment by drashna** [comment]
> Guys i'm having trouble isntalling MSYS and even if i isntall it i have no idea how to easily swap the location of the files. could you please give me a hand?

Are you using QMK MSYS? 
https://msys.qmk.fm/

Also, you can do a lot of stuff from the browser.

**Comment by su8044** [comment]
yep, tried qmk msys too but for some reason i get an error compiling. either it compiles with an older version of qmk, or it doesn't at all. 

how can i make the changes to the file structure from the browser? never tried that

**Comment by drashna** [comment]
if you head to the discord, can probably sort the msys issues for you.  But QMK WSL may work better. 

as for using the web: https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo/PCB_5x12

There is an ellipse button in the top right corner that can delete files.

**Comment by su8044** [comment]
I'll connect to the qmk discord tonight and try again with MSYS. if I'm not successful ill delete that folder. thanks

**Comment by su8044** [comment]
Alright, i finally managed to delete Gerber and dwg folders :)

File: keyboards/nostromo/keymaps/default/readme.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Nostromo is a simple 5x12 layout ortholinear keyboard complete with
+[PCB FILES](https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo/PCB_5x12),
+and
+[CASE FILES](https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo/DWG).
+using a Blackpill F411 MCU and QMK Firmware. Pins are available for RGB, and LED backlight controlled by PWM
+
+if you have any questions please let me know by email. hope you enjoy it!
+
+* Keyboard Maintainer: [su8044](https://github.com/su8044)
+**Email:** su8044@gmail.com
```
Comment (fauxpark) [inline keyboards/nostromo/keymaps/default/readme.md:14]:
```suggestion
```

File: keyboards/preonic/keymaps/default/readme.md [hunk 5]
```diff
@@ -1 +1,20 @@
-# The default Preonic layout - largely based on the Planck's
\ No newline at end of file
+# Nostromo
```
Comment (fauxpark) [inline keyboards/preonic/keymaps/default/readme.md:1]: Please revert this file.

File: keyboards/nostromo/rules.mk [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+ENCODER_ENABLE = no         # Enable rotary encoder support
+AUDIO_ENABLE = no           # Audio output
+WS2812_DRIVER = pwm
+BACKLIGHT_DRIVER = pwm
+LAYOUTS = ortho_5x12
+
```
Comment (fauxpark) [inline keyboards/nostromo/rules.mk:27]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
WS2812_DRIVER = pwm
BACKLIGHT_DRIVER = pwm

LAYOUTS = ortho_5x12
```

File: keyboards/nostromo/readme.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Nostromo is a simple 5x12 layout ortholinear keyboard complete with
+[PCB FILES](https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo/PCB_5x12),
+and
+[CASE FILES](https://github.com/su8044/qmk_firmware/tree/nostromo_maintenance/keyboards/nostromo/DWG).
+using a Blackpill F411 MCU and QMK Firmware. Pins are available for RGB, and LED backlight controlled by PWM
+
+if you have any questions please let me know by email. hope you enjoy it!
+
+* Keyboard Maintainer: [su8044](https://github.com/su8044)
+**Email:** su8044@gmail.com
```
Comment (fauxpark) [inline keyboards/nostromo/readme.md:14]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/nostromo/keymaps/default/readme.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Nostromo
```
Comment (noroadsleft) [inline keyboards/nostromo/keymaps/default/readme.md:1]: This file can actually be removed.

```
git rm keyboards/nostromo/keymaps/default/readme.md
git commit -m "Remove keyboards/nostromo/keymaps/default/readme.md"
```

File: keyboards/preonic/keymaps/default/readme.md (hunk 5 above)
Comment (noroadsleft) [reply keyboards/preonic/keymaps/default/readme.md:1]:
```
git checkout 3fa592a4024a24a636fa0c562e6761667a94f565 -- 3fa592a4024a24a636fa0c562e6761667a94f565
git commit -m "Revert keyboards/preonic/keymaps/default/readme.md"
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**